test/format/bulk.c

```diff
@@ -71,16 +112,25 @@ wts_load(void)
 	key_gen_init(&key);
 	val_gen_init(&value);
 
+	if (g.c_txn_timestamps)
+		bulk_begin_transaction(session);
```

Comment:
I'm a little surprised you can do a bulk load in a timestamp transaction - since I don't think the timestamp will be applied, but I guess it won't hurt to have the transaction running.